### PR TITLE
[PREVIEW] Fixes for respondent step layout

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
# Description

Fixes for respondent step layout
Fixes # 3100

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Locally, unit and e2e testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: resources/mock.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
   "divorceCenterPhone": "0115 955 8266",
   "divorceCenterUrl": "https://courttribunalfinder.service.gov.uk/courts/east-midlands-divorce-unit",
   "respondentName": "Zoe Elizabeth Dante",
-  "petitionerName": "John James Dante"
+  "petitionerName": "John James Dante",
+  "isLoggedIn": "yes"
```
Comment (dtaylor7) [inline resources/mock.json]: whats this for?

Comment (dtaylor7) [reply resources/mock.json]: probably should be based on req.idam?

Comment (dtaylor7) [reply resources/mock.json]: checkout idam-middleware, it adds req.idam if user is loggedin

File: test/unit/middleware/setLocalsMiddleware.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const modulePath = 'middleware/setLocalsMiddleware';
+
+const { idamLoggedin } = require(modulePath);
+const { sinon, expect } = require('@hmcts/one-per-page-test-suite');
+
+describe(modulePath, () => {
```
Comment (tchow8) [inline test/unit/middleware/setLocalsMiddleware.test.js]: Could you have two tests here maybe, one for the success and one for the failing scenario.

**Pull request merged**